src/atoms/TextInput/TextInput.web.js

```diff
@@ -1,3 +1,486 @@
-const TextInput = () => null;
+import React, { useContext, useState, useCallback, useEffect, useRef } from 'react';
+import styled, { ThemeContext } from 'styled-components';
+import PropTypes from 'prop-types';
+import Flex from '../Flex';
+import View from '../View';
+import Space from '../Space';
+import Size from '../Size';
+import isEmpty from '../../_helpers/isEmpty';
+import isDefined from '../../_helpers/isDefined';
+import automation from '../../_helpers/automation-attributes';
+import Label from './Label';
+import CharacterCount from './CharacterCount';
+import AccessoryIcon from './AccessoryIcon';
+import AccessoryText from './AccessoryText';
+import Text from './Text';
+import Line from './Line';
+
+const styles = {
+  textInput: {
+    padding({ variant }) {
+      if (variant === 'filled') {
+        return [0, 1, 0, 1];
+      }
+      return [0];
+    },
+    color({ theme, disabled }) {
+      if (disabled) {
+        return theme.colors.shade[940];
+      }
+      return theme.colors.shade[980];
+    },
+  },
+  fillContainer: {
+    backgroundColor({ variant, isFocused, theme, disabled }) {
+      if (variant === 'outlined') {
+        return 'transparent';
+      } else if (disabled) {
+        return theme.colors.tone[930];
+      } else if (isFocused) {
+        return theme.colors.tone[950];
+      }
+      return theme.colors.tone[930];
+    },
+    hoverBackgroundColor({ variant, theme, disabled }) {
+      if (variant === 'filled') {
+        if (disabled) {
+          return theme.colors.tone[930];
+        }
+        return theme.colors.tone[940];
+      }
+      return '';
+    },
+  },
+  inputContainer: {
+    width({ width }) {
+      switch (width) {
+        case 'auto':
+          return '100%';
+        case 'small':
+          return '160px';
```

Comment:
use px value function.

Reply:
changed on this PR https://github.com/razorpay/blade/pull/160/files